planner/core/planbuilder.go

```diff
@@ -361,6 +364,21 @@ func (b *PlanBuilder) buildSet(v *ast.SetStmt) (Plan, error) {
 	return p, nil
 }
 
+func (b *PlanBuilder) buildCreateBindPlan(v *ast.CreateBindingStmt) (Plan, error) {
+	charSet, collation := b.ctx.GetSessionVars().GetCharsetInfo()
+	p := &SQLBindPlan{
+		BindType:            OpSQLBindCreate,
+		NormalizedOriginSQL: parser.Normalize(v.OriginSel.Text()),
+		BindSQL:             v.HintedSel.Text(),
+		IsGlobal:            v.GlobalScope,
+		BindStmt:            v.HintedSel,
+		Charset:             charSet,
+		Collation:           collation,
+	}
+	b.visitInfo = appendVisitInfo(b.visitInfo, mysql.SuperPriv, "", "", "", nil)
```

Comment:
`nil` error would cause a `privilege check fail` error message, which is not friendly to the user. I think we can construct a specific error message for this scenario.

Reply:
TIDB has many same code.
![image](https://user-images.githubusercontent.com/21154201/56199433-7b6eb500-606f-11e9-88b1-12e2a5cb79d2.png)